ddtrace/internal/agent.py

```diff
@@ -9,23 +9,42 @@
 from .uds import UDSHTTPConnection
 
 
-DEFAULT_HOSTNAME = "localhost"
-DEFAULT_TRACE_PORT = 8126
+DEFAULT_HTTP_HOSTNAME = "localhost"
+DEFAULT_UNIX_HOSTNAME = "unix"
+DEFAULT_HTTP_TRACE_PORT = 8126
+DEFAULT_UNIX_TRACE_PORT = "///var/run/datadog/dsd.socket"
```

Comment:
This is the default socket for Dogstatsd, we need to be defaulting tracing for `/var/run/datadog/apm.socket`

We then also need to figure out how to default to the `dsd.socket` one for Dogstatsd

Reply:
Should the Dogstatsd socket change be in a separate PR or the same?

Reply:
Same could be ok. Depends on how complex the PR becomes. That is a good way to make the PR smaller, but logically makes sense to be in the same PR